apps/explorer/README.md

```diff
@@ -0,0 +1,35 @@
+# Explorer
```

Comment:
should this file be liked from the main README.md?

Reply:
Makes sense! Added it